src/main/java/com/tabnine/inline/CompletionAdjustment.java

```diff
@@ -0,0 +1,12 @@
+package com.tabnine.inline;
+
+import com.tabnine.binary.requests.autocomplete.AutocompleteRequest;
+import com.tabnine.binary.requests.autocomplete.AutocompleteResponse;
+
+public interface CompletionAdjustment {
+  void adjustRequest(AutocompleteRequest autocompleteRequest);
```

Comment:
mmm i prefer the signature to be `AutocompleteRequest adjustRequest(AutocompleteRequest autocompleteRequest);` - such that the function takes an object and returns a processed object, instead of mutating it internally. its less side-effect-y this way